### ISSUE TITLE
TPS lag AE2 nested crafting

### ISSUE THREAD

**Issue by blichte1** [Cazadorsniper/PO3#1011]
Pack Version: 3.2.55

Pack Mode: Titan

Singleplayer, Multiplayer, or LAN?: Singleplayer

Description: When crafting Mekanism's Ultimate Induction Cells, TPS lag spikes (sky jumps, other machines slow). This is reproducible: it seems to be coming from the AE2 assembly multiblock (or at least it's coding) as providing the full resources for the autocrafting yields the same lag. Notably it gets worse the longer the autocrafting goes on (so calling up more induction cells yields worse and worse lag).

Steps to reproduce: Construct/use AE2 system to automate the assembly of complex items with nested crafting steps (ie Ultimate Induction Cells).

Crash log (use pastebin or issue will be closed): None. The lag resolves itself after a few minutes.

**Comment by Cazadorsniper** [comment]
I can not fix this.

**Comment by blichte1** [comment]
Do you suggest I report this to the AE2 github?

**Comment by Cazadorsniper** [comment]
You can.

**Comment by tawraven** [comment]
I found that AE2 autocrafting does not work well with a wall of drawers/barrels/etc. connected behind a storage bus.  Apparently it takes a long time to traverse thru them to find ingredients.  I ended up making max size storage disks and importing everything into ME system.  This makes autocrafting much faster.  If you have a similar setup, disconnect the drawers from your system and see if lag goes away.
